test/run

```diff
@@ -53,7 +68,7 @@ def collapse(string):
     return re.sub(r"\s+", " ", string)
 
 
-def add_e2e_tests(litani, test_dir, root_dir, counter):
+def add_e2e_tests(litani, test_dir, root_dir, counter, output_dir=None):
```

Comment:
`output_dir` should be part of this function's `*args` and not be part of  `**kwargs`.

Reply:
Resolved